src/libs/uploads.ts

```diff
@@ -11,7 +11,7 @@ export type UploadState = {
   currentFile: File | null;
   files: File[];
   completedFiles: File[];
-  errors: unknown[];
+  errors: { file: File; error: unknown }[];
```

Comment:
Store the file associated with the error so that we can put file information in the error notification.